xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
I don't know if "should" is the right word here, or if it should only be done when first connecting.